pkg/api/core/v1/client/namespaces.go

```diff
@@ -22,33 +16,7 @@ func (c *Client) NamespaceCreate(req models.NamespaceCreateRequest) (models.Resp
 		return resp, nil
```

Comment:
It's not related to the PR but should be this returning `err` instead of `nil`?

Reply:
Quite possible. I will make that change and see if that breaks CI.

Reply:
Lets see what CI says about ea54e449a52827369fb0c5fa3158ea06622bd4d8